service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/ContactPointMapper.java

```diff
@@ -0,0 +1,44 @@
+package uk.nhs.adaptors.oneoneone.cda.report.mapper;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import uk.nhs.connect.iucds.cda.ucr.TEL;
+import org.hl7.fhir.dstu3.model.ContactPoint;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ContactPointMapper {
+
+    @Autowired
+    PeriodMapper periodMapper;
```

Comment:
See Java coding standards. Let's try to use constructors instead of field injection.
Field should be private